pkg/api/openapi/minder/v1/minder.swagger.json

```diff
@@ -2396,6 +2403,57 @@
           "UserService"
         ]
       }
+    },
+    "/api/v1/user/invitation/{nonce}": {
+      "post": {
+        "summary": "ResolveInvitation allows a user to accept or decline an\ninvitation to a project given the nonce for the invitation.\nA user may call ResolveInvitation to accept or decline an\ninvitation even if they have not called CreateUser.  If a\nuser accepts an invitation via this call before calling\nCreateUser, a Minder user record will be created, but no\nadditional projects will be created (unlike CreateUser,\nwhich will also create a default project).",
+        "operationId": "UserService_ResolveInvitation",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/v1ResolveInvitationResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "nonce",
+            "description": "nonce is the nonce of the invitation to resolve.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "accept",
+            "description": "accept is true if the invitation is accepted, false if it is rejected.",
```

Comment:
hmm, do we actually have a use case for rejecting an invitation? :thinking: if I receive an invitation for something that I don't recognize/was not intended for me I will probably not click any links in the email :thinking: 

but I guess this is not something that would negatively affect FE

Reply:
The invitation may also show up in the invite list (presented in the UI or as a prompt on the CLI) if it's sent to the primary email address associated with your account.  You might want to be able to dismiss that invitation if you think it's junk.

Reply:
true